browser/net/brave_site_hacks_network_delegate_helper_browsertest.cc

```diff
@@ -370,6 +421,73 @@ IN_PROC_BROWSER_TEST_F(BraveSiteHacksNetworkDelegateBrowserTest,
   }
 }
 
+IN_PROC_BROWSER_TEST_F(BraveSiteHacksNetworkDelegateBrowserTest,
+                       QueryStringFilterTwitterServiceWorker) {
+  const std::string inputs[] = {
+      "t=1234",
+      "t=12345&foo=bar",
+      "value=yes&t=123456",
+  };
+  const std::string outputs[] = {
+      "",
+      "foo=bar",
+      "value=yes",
+  };
+
+  constexpr size_t input_count = std::size(inputs);
+  static_assert(input_count == std::size(outputs),
+                "Inputs and outputs must have the same number of elements.");
```

Comment:
this can be simplified with something like:
```
struct TestCase {
  const char* input;
  const char* output;
} const kTestCases[] = {
  {"t=1234", ""},
  ...
};

for (const auto& test_case : kTestCases) {
  // Add some test case context on failure.
  SCOPED_TRACE(testing::Message() << test_case.input << " " << test_case.output);
  ...
}
```

Reply:
Very nice!